### PR TITLE
LG-9314 Sample apps should have failure_to_proof URLs

### PR DESCRIPTION
**Pull request by amirbey**
[LG-9314](https://cm-jira.usa.gov/browse/LG-9314)

### What
Create a route and page for failure to proof
saml-sinatra-failure-to-proof

Note: this page has recently been added for the [identity-oidc-sinatra](https://github.com/18F/identity-oidc-sinatra) app

### CODE REVIEW

File: views/failure_to_proof.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="clearfix">
+  <div class="col-12 sm-col-6 mx-auto">
+    <div class="h5 p2 center">
+      <span class="bold caps">Failure to Proof</span>
+      <p><strong>We were unable to verify your identity</strong>.</p>
+    </div>
+  </div>
+</div>
```
Comment (aduth) [inline views/failure_to_proof.erb:8]: Could we use the same markup as in the OIDC implementation in https://github.com/18F/identity-oidc-sinatra/pull/141 ? Not sure if this is meant to be internally consistent with other SAML views, but I think it'd make sense for the two sample apps to look the same, and these BassCSS classes (`mx-auto`, etc.) are legacy and undesirable vs. design system.

```suggestion
<h1>Failure to Proof</h1>
<div>We were unable to verify your identity.</div>
```

File: app.rb [hunk 2]
```diff
@@ -148,6 +148,11 @@ def get_param(key, acceptable_values)
     end
   end
 
+  get '/failure_to_proof' do
+    puts 'Failure to Proof :('
+    erb :failure_to_proof#, layout: true
```
Comment (aduth) [inline app.rb:153]: Can we remove the commented code?

```suggestion
    erb :failure_to_proof
```

**Review by aduth — APPROVED** [review]
LGTM 👍

File: views/failure_to_proof.erb (hunk 1 above)
Comment (amirbey) [reply views/failure_to_proof.erb:8]: hi @aduth  ... yes, I was attempting to be consistent with the other saml views.  I am also good with your suggestion to be consistent witht the oidc app.  accepted ... thanks!
Screen Shot 2023-04-28 at 11 20 26 AM

**Pull request merged**